cleverhans/utils.py

```diff
@@ -113,6 +118,20 @@ def other_classes(nb_classes, class_ind):
     return other_classes_list
 
 
+def conv_wrap(filters, kernel, strides, padding):
```

Comment:
@npapernot After changing the small bug on version check it works on my machine.